core/cautils/fileutils.go

```diff
@@ -325,15 +326,14 @@ func GetFileFormat(filePath string) FileFormat {
 }
 
 // handleListObject handle a List manifest
-func handleListObject(obj workloadinterface.IMetadata) []workloadinterface.IMetadata {
+func handleListObject(list *listworkloads.ListWorkloads) []workloadinterface.IMetadata {
+
 	yamlObjs := []workloadinterface.IMetadata{}
-	if i, ok := workloadinterface.InspectMap(obj.GetObject(), "items"); ok && i != nil {
-		if items, ok := i.([]interface{}); ok && items != nil {
-			for item := range items {
-				if m, ok := items[item].(map[string]interface{}); ok && m != nil {
-					if o := objectsenvelopes.NewObject(m); o != nil {
-						yamlObjs = append(yamlObjs, o)
-					}
+	if items := list.GetItems(); items != nil {
+		for item := range items {
+			if m, ok := items[item].(map[string]interface{}); ok && m != nil {
```

Comment:
What are the items types?
Why are they not `map[string]inteface`

Reply:
Because the newObject function produces Imetadata obj which is map[string]interface{} so the items inside the list is a list of interface, so I have to treat it like that.